### ISSUE TITLE
2020-03-16 Data: Separate entries for "Congo (Brazzaville)" and  "Republic of the Congo"

### ISSUE THREAD

**Issue by plt005** [CSSEGISandData/COVID-19#893]
Aren't these the same countries? (Not to be confused with the Democratic Republic of Congo).

**Comment by KenRandall** [comment]
I agree. You now have 3 entries for the Congo when there should only be 2!

**Comment by elitemaks** [comment]
This repo reveals several issues with geo names and their standardization. In most cases, there is no need to use official names, like the official name for Italy is "Italian Republic". There is only one Italy and it's easier to use a widely used name. 
As for Congo, there are 2 countries with the same short name, but different official names. In this case, I'd suggest using official names:
1. "Republic of the Congo" - also referred here as "Congo-Brazzaville"
2. "Democratic Republic of the Congo" - referred as "Congo (Kinshasa)"
